node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeRepository.java

```diff
@@ -213,15 +208,30 @@ private void rewriteNodes() {
     /** Returns this node repo's view of the applications deployed to it */
     public Applications applications() { return applications; }
 
-    public NodeFlavors flavors() {
-        return flavors;
-    }
+    /** Returns the load balancers available in this node repo */
+    public LoadBalancers loadBalancers() { return loadBalancers; }
+
+    public NodeFlavors flavors() { return flavors; }
 
     public HostResourcesCalculator resourcesCalculator() { return resourcesCalculator; }
 
     /** The number of nodes we should ensure has free capacity for node failures whenever possible */
     public int spareCount() { return spareCount; }
 
+    /**
+     * Returns ACLs for the children of the given host.
+     *
+     * @param host node for which to generate ACLs
+     * @return the list of node ACLs
+     */
+    public List<NodeAcl> getChildAcls(Node host) {
+        if ( host.type() != NodeType.host) throw new IllegalArgumentException("Only hosts have children");
```

Comment:
Should check `isHost()` instead.

Reply:
Done -  thanks.  I see we have many other NodeType.host comparisons as well. I guess all should likely use isHost(), or have you looked at that before?

Reply:
Yes, generally we should use `isHost()`, but I think there is a lot of code that only applies to tenant hosts.

I did a quick check on usages in this module, and looks like `NodeType.host` is mostly used in dynamic provisioning, capacity decisions related to node allocation and rebalancing. I.e. cases where you really mean `NodeType.host`.